BeagleBone_black/src/data/data.hpp

```diff
@@ -188,18 +197,30 @@ class Data {
    */
   void setMotorData(const Motors& motor_data);
 
+  /**
+     * @brief      Retrieves data on whether stop/kill power commands have been issued.
+     */
+    Communications getCommunicationsData();
+
+    /**
+     * @brief      Should be called to update motor data.
+     */
+    void setCommunicationsData(const Communications& communications_data);
+
  private:
   StateMachine state_machine_;
   Navigation  navigation_;
   Sensors     sensors_;
   Motors      motors_;
   Batteries   batteries_;
+  Communications      communications_;
```

Comment:
No point in having multiple spaces here. Make it just one and possibly make the lines above aligned.